presto/datadog_checks/presto/data/conf.yaml.example

```diff
@@ -35,15 +35,15 @@ instances:
   #
   #  tags:
   #    - environment: <NODE_ENVIRONMENT>
-  #    - node_type: <NODE_TYPE>   # e.g. coordinator or worker
+  #    - node_type: <NODE_TYPE>
```

Comment:
Shouldn't this be like all the other integrations ? 
```  
  #  tags:
  #    - <KEY_1>:<VALUE_1>
  #    - <KEY_2>:<VALUE_2>
```

Reply:
you are right, I fixed it.